### PR TITLE
Add UDS Alert Banner

### LINKED ISSUE #30: Global Alert Banner
We should add a notification area (in the form of a web-standards compliant banner) to all templates. This area will be for when we might need a global alert or notification across a site - such as the recent need for a COVID banner. Initial requirements would include:

- A widget area just below the hero image, and above all other content, that spans the full content width (?)
- A widget that would allow the user to choose the banner settings:
  - Icon (optional)
  - Title text
  - Content text
  - Up to two button links with:
  - button text
  - URL

### PR DESCRIPTION
**Pull request by wmcconnell**
Resolves #30 

Adds a widget area, and corresponding widget, for displaying a UDS banner across the top of the content area and just beneath the hero. Allows the user to choose:

- background color
- icon
- title and body text
- up to two buttons (which is based on the current example in Storybook)

Changes proposed in this pull request:

- A new widget area on `page.php` for displaying the widget
- A new widget, the *Notification Banner*, for displaying widget content.
- Advanced Custom Fields back-end interface
- A new section in the README file with basic  instructions for use

This is an MVP solution just to close that issue and provide an implementation that matches current Storybook. There are some possible improvements, and I would welcome suggestions.

### CODE REVIEW

File: functions.php [hunk 1]
```diff
@@ -9,27 +9,28 @@
...
+	'/customizer.php',                           // Customizer additions.
+	'/custom-comments.php',                      // Custom Comments file.
+	'/jetpack.php',                              // Load Jetpack compatibility file.
+	'/wp-custom-fields.php',                     // Disable WP Core custom fields metaboxes.
+	'/tgm-plugin-activation.php',                // Load TGM Plugin Activation script for dependent plugin recommendations.
+	'/class-wp-social-media-walker.php',         // Load custom WordPress nav walker for footer Social Media menu wdiget.
+	'/wp-custom-menu.php',                       // Load custom menu builder functions.
+	'/editor.php',                               // Load Editor functions.
+	'/deprecated.php',                           // Load deprecated functions.
+	'/class-uds-notification-banner.php', // Load the UDS notification widget.
```
Comment (nathan-rollins) [inline functions.php:33]: There isn't a hard rule about the order of these includes, but there is a little logic. Early bootstrap logic when the theme is initializing and setup is early i this list and the load order should somewhat make sense in terms of dependencies or overrides.

Your new include file, "class-uds-notification-banner.php" is setting up a new widget area, it should probably load following the widgets.php include.

**Review by nathan-rollins — COMMENTED** [review]
Haven't we already reviewed the code in 'asu-navigation-menu.php' in a different Pull Request? What is happening here? Is this because this is a child branch from another branch still in PR review? 

If so, then we need to stop this practice and make sure everyone understands that each new feature branch they create MUST be based off the latest version of the develop branch. Until that other PR is merged into develop, this PR is showing code changes not relevant to this specific branch.

**Comment by nathan-rollins** [comment]
Yep, this branch is built off the changes in PR #63.  Of course, that means this PR cannot be approved and merged into develop before the other PR is resolved and merged in. (And if for some reason that other PR had to be aborted, this PR would also have to be aborted.)

File: functions.php (hunk 1 above)
Comment (wmcconnell) [reply functions.php]: Pushed a new commit to move that include directly underneath 'widgets.php'

**Comment by wmcconnell** [comment]
I'm going to go ahead and merge this, as it's been waiting long enough, and I want to clear the way as much as possible for brining in Steve and Zainab's work.

**Pull request merged**